03_launch_mgmt_cluster.sh

```diff
@@ -79,6 +79,20 @@ function launch_minikube() {
     minikube start
 }
 
+function launch_kinder() {
+  kinder="${GOPATH}/bin/kinder"
+  if [[ $(sudo $kinder get clusters) == "" ]]; then
+    sudo $kinder create cluster
+    sudo $kinder do kubeadm-init
+    kinder_kubeconfig_path="$(sudo $kinder get kubeconfig-path)"
+    sudo rm -f ${HOME}/.kube/config
+    mkdir -p ${HOME}/.kube/
+    sudo mv $kinder_kubeconfig_path ${HOME}/.kube/config
+    sudo chown ${USER}:${USER} ${HOME}/.kube/config
+    #TODO: Attach `provisioning` interface to node which runs Ironic service.
```

Comment:
not sure I understand the TODO here -- Ironic is running on the host where the `provisioning` bridge is located -- can you expand on what's left to do here?

Reply:
Sorry, my mistake. 
Previously, I thought that if we move Ironic to run inside a container, we will need `provisioning` bridge to make it visible to the network outside the cluster. But it seems to be my misunderstanding.